pint/fitter.py

```diff
@@ -50,16 +50,13 @@ def set_params(self, fitp):
         for p, val in zip(fitp.keys(), fitp.values()):
             # If value is unitless but model parameter is not, interpret the
             # value as being in the same units as the model
-            modval = getattr(self.model, p).value
+            modval = getattr(self.model, p).num_value
             # Right now while the code below preserves the unit, Angle types
```

Comment:
You should probably remove the comment(s) that are not applicable anymore since the Angle stuff has been removed.